packages/gluestick/templates/new/src/config/__tests__/_entry.test.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import { Route } from 'react-router';
-import { shallow } from 'enzyme';
-import { getHttpClient } from 'gluestick-shared';
+import React from "react";
```

Comment:
Yup, this is because the `.eslintrc` from `templates`. We should create an issue to refactor it.